packages/runtime/src/interfaces/response.ts

```diff
@@ -60,4 +60,66 @@ export type HttpStatusCodeLiteral =
   | 510
   | 511;
 
+export type HttpStatusCodeLiteralStr =
```

Comment:
Let's use a template literal type here: 

```ts
type HttpStatusCodeString = `${HttpStatusCode}`;
```

Reply:
Didn't realize it would work like this 😅